ci/lava/tests/test-minimal-production.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Pytest for testing a minimal rootfs on production.
+
+It does this by checking that programs or services are not available.
+"""
+
+import pytest
+import re
+
+
+"""Commands not checked for as installed by other packages
+- see JIRA IOTMBL-2381
+    "blkid",  # util-linux installed
+    "depmod",  # kmod installed
+    "fsck",  # util-linux installed
+    "getty",  # agetty installed
+    "groups",  # shadow installed
+    "insmod",  # kmod installed
+    "login",  # shadow installed
+    "lsmod",  # kmod installed
+    "modprobe",  # kmod installed
+    "mount",  # util-linux installed
+    "passwd",  # shadow installed
+    "reboot",  # systemctl installed
+    "rmmod",  # kmod installed
+    "su",  # shadow installed
+    "sulogin",  # util-linux installed
+    "umount",  # util-linux installed
+    "fsck.ext2",  # systemd installed ext2 feature from e2fsprogs
+    "mkfs.ext2",  # cloud-client installed ext2 feature from e2fsprogs
+
+Features not checked for as no obvious extra packages installed
+    keyboard
+    screen
+    touchscreen
+"""
+
+
+class TestRunner:
```

Comment:
Make this a more specific class name. The name goes into the results database so it will make it easier to identify them.